relayer/azero_client/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "azero_client"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+subxt = {  workspace = true, features = ["unstable-reconnecting-rpc-client", "substrate-compat"] }
+thiserror = "1.0.63"
+log = "0.4.20"
+async-trait = "0.1.81"
+tokio = "1.40.0"
+anyhow = "1.0.79"
+serde = "1.0.209"
+contract-transcode = "3.2.0"
+pallet-contracts-primitives = "29.0.0"
```

Comment:
This one crate is a `polkadot-sdk` dependency. Ideally it should be compiled from our fork, ie
```
pallet-contracts-primitives = { git = "https://github.com/Cardinal-Cryptography/polkadot-sdk.git", branch = "aleph-v1.2.0" } 
```
But there's some problem with this approach already. It seems to be removed now in 1.6.0 version: https://github.com/paritytech/polkadot-sdk/pull/2806. This is because it was moved to the `pallet-contracts` create itself (file `substrate/frame/contracts/src/primitives.rs`). Even we use that only symbol `pallet_contracts_primitives::ContractExecResult`, I think it's better we use the version from `pallet-contracts`. Yes this means we we'd depend here on `pallet-contracts = { git = "https://github.com/Cardinal-Cryptography/polkadot-sdk.git", branch = "aleph-vX.Y.Z" }`, whereas `aleph-vX.Y.Z` is `aleph-node` `main` branch Cargo.toml ,but I'm afraid is unavoidable.

Reply:
you are right, done